Documentation/platforms/arm/rp2040/boards/seeed-xiao-rp2040/README.txt

```diff
@@ -0,0 +1,96 @@
+README
+======
+
+This directory contains the port of NuttX to the Seeed Studio Xiao RP2040.
+See ttps://wiki.seeedstudio.com/XIAO-RP2040/ for information about Seeed
```

Comment:
```suggestion
See https://wiki.seeedstudio.com/XIAO-RP2040/ for information about Seeed
```

Reply:
Hi @hartmannathan ,
It has been fixed.
Thanks